crates/cli/cli/Cargo.toml

```diff
@@ -32,3 +32,4 @@ oneline-eyre              = { workspace = true }
 humantime                 = { workspace = true }
 tracing                   = { workspace = true }
 serde_json                = { workspace = true }
+serde = { version = "1.0.196", features = ["derive"] }
```

Comment:
Common dependencies can be specified in the workspace Cargo.toml file. So here we can specify the dependency as  `serde = { workspace = true, features = ["derive"] }`.

Reply:
3d2fa39a23d14ea5c53290c20f43e47eead2138e